xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1161,6 +1164,14 @@ void DX::DeviceResources::CheckDXVA2SharedDecoderSurfaces()
 
   CLog::LogF(LOGINFO, "DXVA2 shared decoder surfaces is{}supported",
              m_DXVA2SharedDecoderSurfaces ? " " : " NOT ");
+
+  m_DXVASuperResolutionSupport =
+      m_d3dFeatureLevel >= D3D_FEATURE_LEVEL_12_1 &&
+      ((ad.VendorId == PCIV_Intel && driver.valid && driver.majorVersion >= 31) ||
```

Comment:
Requirements seem less stringent for Intel - Chrome enables for Intel Gen9+, drivers above 27.20.100.8681?
I thought you had a gen 12, maybe you could try it.
I should have a gen 13 in a week or two to test as well.

Reply:
Already tested in Intel 13th gen i7-13700K / UHD graphics 770:

Intel enable code works and even graphics usage increases but not appreciable difference in output picture. 

My reasoning is that although they share same video driver, Arc 750 / 770 is needed since the integrated graphics have only a fraction of the power needed. For this reason is mentioned in help text that only works in NVIDIA RTX 40x, 30x and Intel Arc 770, 750.